doc/source/cheatsheet/cheatsheet.qmd

```diff
@@ -41,23 +41,22 @@ jupyter:
 
 ## Local connection to a Workbench Server
 
-Execute the steps below to link PyWorkbench with a local Ansys Workbench session:
+Execute the steps below to link PyWorkbench with a local session:
```

Comment:
```suggestion
Perform these steps to link PyWorkbench with a local session:
```
Google dev style guide says to avoid use of page location descriptors (like above and below) in content.

Reply:
Thanks!